### PR TITLE
Rewrite scan20 overview

### PR DESCRIPTION
**Pull request by ryan-a-baker**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
1.6 

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: scst-scan/app-scanning-beta.hbs.md [hunk 1]
```diff
@@ -10,24 +10,19 @@ This topic tells you how to install and configure Supply Chain Security Tools (S
 
 ## <a id="overview"></a>Overview
 
-SCST - Scan 2.0 is responsible for providing the framework to scan applications
-for their security posture. Scanning container images for known Common
-Vulnerabilities and Exposures (CVEs) implements this framework. This framework
-simplifies integration for new plug-ins by allowing users to integrate new scan
-engines and minimize the scope of the scan engine to only scan and push results
-to an OCI compliant registry.
+SCST - Scan 2.0 is responsible for providing the framework to scan components of workloads to help users increase the security posture of their application.  The initial capability for this component is to provide the ability to scan container images using a users prefered container image scan solution for known Common Vulnerabilities and Exposures (CVEs).  By including this component in a users software supply chain, a user can identify vulnerabilites earlier in the development lifecycle.
```
Comment (rxl7906) [inline scst-scan/app-scanning-beta.hbs.md:13]:
```suggestion
The initial capability for this component is to provide the ability to scan container images using a users preferred container image scan solution for known Common Vulnerabilities and Exposures (CVEs).  By including this component in a users software supply chain, a user can identify vulnerabilities earlier in the development lifecycle. 
```

File: scst-scan/app-scanning-beta.hbs.md [hunk 2]
```diff
@@ -10,24 +10,19 @@ This topic tells you how to install and configure Supply Chain Security Tools (S
...
 
-SCST - Scan 2.0 is responsible for providing the framework to scan applications
-for their security posture. Scanning container images for known Common
-Vulnerabilities and Exposures (CVEs) implements this framework. This framework
-simplifies integration for new plug-ins by allowing users to integrate new scan
-engines and minimize the scope of the scan engine to only scan and push results
-to an OCI compliant registry.
+SCST - Scan 2.0 is responsible for providing the framework to scan components of workloads to help users increase the security posture of their application.  The initial capability for this component is to provide the ability to scan container images using a users prefered container image scan solution for known Common Vulnerabilities and Exposures (CVEs).  By including this component in a users software supply chain, a user can identify vulnerabilites earlier in the development lifecycle. 
+ 
+The Scan 2.0 simplifies integrating image scan solutions such as Anchore's Grype, Aqua's Trivy, Palo Alto's Prisma, and VMware's Carbon Black Cloud in to the Tanzu Application Platform.  This is achieved by the following enhancements over Scan 1.0:
```
Comment (rxl7906) [inline scst-scan/app-scanning-beta.hbs.md:15]:
```suggestion
The Scan 2.0 simplifies integrating image scan solutions such as Anchore's Grype, Aqua's Trivy, Palo Alto's Prisma, and VMware's Carbon Black Cloud into the Tanzu Application Platform. This is achieved by the following enhancements over Scan 1.0:
```

File: scst-scan/app-scanning-beta.hbs.md [hunk 3]
```diff
@@ -10,24 +10,19 @@ This topic tells you how to install and configure Supply Chain Security Tools (S
...
 
-During scanning:
+* Reducing the scope of what image scanning is resposible to only be 1) scanning a container image and 2) pushing the scan results in an industry standard format ([CycloneDX](https://cyclonedx.org/) or [SPDX](https://spdx.dev/) as an industry standard OCI artifact to an OCI compliant container registry.  Using industry standards mitigates the need for users wishing to create an integration to be familar with properitary Tanzu Application Platform configurations.
```
Comment (rxl7906) [inline scst-scan/app-scanning-beta.hbs.md:17]:
```suggestion
* Reducing the scope of what image scanning is responsible to only be:
** Scanning a container image.
** Pushing the scan results in an industry standard format ([CycloneDX](https://cyclonedx.org/) or [SPDX](https://spdx.dev/) as an industry standard OCI artifact to an OCI compliant container registry.  Using industry standards mitigates the need for users wishing to create an integration to be familiar with proprietary Tanzu Application Platform configurations.
```

**Comment by ryan-a-baker** [comment]
Thanks for the review @rxl7906 - opened a separate [PR](https://github.com/pivotal/docs-tap/pull/2723/files) to fix recommendations.

**Pull request merged**